VirtualReality/Widgets/qMRMLVirtualRealityHomeWidget.cxx

```diff
@@ -106,6 +113,57 @@ vtkMRMLVirtualRealityViewNode* qMRMLVirtualRealityHomeWidget::virtualRealityView
   return d->VirtualRealityViewNode;
 }
 
+//-----------------------------------------------------------------------------
+void qMRMLVirtualRealityHomeWidget::addModuleButton(QWidget* widget, QIcon* icon)
+{
+  Q_D(qMRMLVirtualRealityHomeWidget);
+
+  QPushButton* moduleButton = new QPushButton(d->ModulesGroupBox);  
+  d->ModulesGroupBoxLayout->addWidget(moduleButton);
+  moduleButton->setIcon(*icon);
+  
+  widget->setParent(d->ModuleWidgetFrame);
+  d->ModuleWidgetsMap[moduleButton] = widget;
+
+  QObject::connect(moduleButton, SIGNAL(clicked()), this, SLOT(onModuleButtonPressed()));
+  QObject::connect(d->backButton, SIGNAL(clicked()), this, SLOT(onBackButtonPressed()));
+ 
+  moduleButton->setVisible(true);
+  widget->setVisible(false);
```

Comment:
Please rename widget to moduleWidget everywhere (including the function declaration in header) for clarity